models/star/Cargo.toml

```diff
@@ -8,3 +8,6 @@ crate-type = ["cdylib"]
 
 [dependencies.fj]
 path = "../../crates/fj"
+
+[dependencies.fj-proc]
+path = "../../crates/fj-proc"
```

Comment:
I think it should be possible, to re-export `fj-proc` from `fj` (`pub use fj_proc;`). Then we wouldn't need the extra dependency, and would also have a simpler syntax in the models (`fj::model` instead of `fj_proc::model`).